laterpay/__init__.py

```diff
@@ -66,7 +66,8 @@ class ItemDefinition(object):
     For Single item purchases: http://docs.laterpay.net/platform/dialogs/buy/
     """
 
-    def __init__(self, item_id, pricing, url, title, expiry=None, sub_id=None, period=None, item_type=None):
+    def __init__(self, item_id, pricing, url, title, expiry=None, sub_id=None,
+                 period=None, election_id=None, item_type=None):
```

Comment:
I'm moving election_id to the end to preserve the ordering of the existing parameters. Even if they are all named arguments.